1_space_odyssey/js/jorn-9000/app.js

```diff
@@ -0,0 +1,154 @@
+       2001: A Space Odyssey spans from the dawn of man to its title year. 
+             Stone Age Earth: 
+             In the presence of a mysterious black obelisk, pre-humans discover the use 
+             of tools -- and weapons -- violently taking first steps toward intelligence. 
+             
+        1999: On Earth's moon astronauts uncover another mysterious black obelisk. 
+        
+        2001: Between Earth and Jupiter, the spacecraft's intelligent computer makes a mistake that 
+              kills most of the human crew -- then continues to kill to hide its error. 
+              
+        2020: The sole survivor of the journey to Jupiter ascends to the next level of humanity.
+ */
+
+
+/*
+  You are the sole survivor who will make the journey to Jupiter.
+
+  But before you can leave you must build an autonomous humanoid robot to help you on your mission.
+
+  This humanoid will be used to perform dangerous and mission critical tasks on your journey to Jupiter.
+  - space walks to repair the spaceship
+  - fight off alien creatures
+  -  
+
+  The humanoid is your doppelgänger, a sentient being that will help you save man from extinction by keeping you alive.
+
+  Using JavaScript build the software that will allow us to describe and programme the humanoid to do our bidding.
+  */
+
+/* ______   __  __   ______  ______   
+        _..._
+      .'     '.      _
+     /    .-""-\   _/ \
+   .-|   /:.   |  |   |
+   |  \  |:.   /.-'-./
+   | .-'-;:__.'    =/
+   .'=  *=|Noroff_.='
+  /   _.  |    ;
+ ;-.-'|    \   |
+/   | \    _\  _\
+\__/'._;.  ==' ==\
+         \    \   |
+         /    /   /
+         /-._/-._/
+  ...    \   `\  \
+          `-._/._/
+
+
+ 1. CREATE DROID::::::
+    Below are all the variables needed to initialize an android into consciousness in its default state.
+    a. assign the correct values to the variables below, 
+    b. ensure to choose the correct values and datatypes
+    c. the droid should be built in your likeness (a human, with human features, 2 arms, legs etc)
+*/
+var isAlive = true;
+var name = "Jorn-9000";
+var hasJetPack = false;
+var suitColor = "red";
+var eyes = 2;
+var eyeColor = "brown";
+var hairColor = "blond";
+var legs = 2;
+var arms = 2;
+var hands = 2;
+var ears = 2;
+var canWalk = true;
+var canRun = true;
+var canFly = false;
+var hasWeapon = false;
+var isInGoodMood = true;
+
+console.log(".-.-. Booting .-.-.");
+console.log(name, isAlive, hasJetPack, eyes, eyeColor, hairColor, legs, arms, hands, ears, canWalk, canFly, hasWeapon);
+
+
+
+/* 
+ 2. BROKEN SATELLITE::::::
+    You need to send the droid out on a space walk to fix the satellite, with out it you can't communicate with earth.
+    a) write an if condition checking if the jet pack is enabled, if true the droid can go outside if false the droid 
+       cannot go outside, console log the correct log based on that condition. 
+*/
+if(hasJetPack)
+{
```

Comment:
please format / indent code 

````JS
if (someThing) {
   console.log("hello");
} else {
    console.log("good bye");
}
````
as per: https://github.com/mannuelf/clean-code-javascript 
have a look how all the if blocks are formatted throughout that document as a guide.